### DISCUSSION TITLE
Email alerts

### DISCUSSION THREAD

**Discussion by c-michail** [TechnitiumSoftware/DnsServer#986] [Ideas]
Hello,
  It would be nice to have somekind of e-mail alerts. eg if the Query per minute are over 30k or something.
  
  Thank you

**Comment by ShreyasZare** [comment]
Thanks for the feature request. Please post this in the Issues section so that it gets tracked. For this feature, there needs to be some more parameters to be added to monitor so as to make it more useful.
